### PR TITLE
Updated Readme.md ; added npm install command in the readme.md

### PR DESCRIPTION
**Pull request by Jeevan-Kiran-Lenka**
**Issue Number**
#62, #72
fixes #

 `npm install`

**Describe the changes you've made**
Changed step 1 of Running in the deployment mode from `npm start` to `npm install`, which will go ahead and download all the dependencies that your project would be required, also changed deployment mode to `Development mode` 

I have added a step of `npm install` in *Running in deployment server*

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**
Can test it by following the readme procedure for setting up the dev environment

A good test plan should give instructions that someone else can easily follow.

If following the readme sets your environment perfectly then the code is working

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Jeevan-Kiran-Lenka** [comment]
@KeenWarrior @Abhishek-kumar09 please review my PR and lemme know

File: README.md [hunk 1]
```diff
@@ -87,10 +87,11 @@ Whenever you want to contribute to any project. It is best practice that you sho
 
 ### 🚩 Running in Deployment mode :
```
Comment (kaiwalyakoparkar) [inline README.md:88]: Change deployment to development

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jeevan-Kiran-Lenka) [reply README.md:88]: Okay on it

**Review by Jeevan-Kiran-Lenka — COMMENTED** [review]
I have reviewed my changes

File: README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Whenever you want to contribute to any project. It is best practice that you sho
 1. Open this clonned folder in text editor of your choice
 2. If you want to use the project using `npm` then that comes along side when you download and install node js
 
-### 🚩 Running in Deployment mode :
+### 🚩 Running in Development mode :
 
 1. Open the terminal and type in `npm install`, this will go on install all the dependicies you need
```
Comment (kaiwalyakoparkar) [inline README.md:90]: Please make it "this will go on and install all the dependencies you need"

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by Jeevan-Kiran-Lenka** [comment]
@Abhishek-kumar09 Please review it now

File: README.md (hunk 2 above)
Comment (kaiwalyakoparkar) [reply README.md:90]: Make this change...👆🏻

Comment (Jeevan-Kiran-Lenka) [reply README.md]: I have made it

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Comment by Jeevan-Kiran-Lenka** [comment]
@kaiwalyakoparkar What's the meaning of LGTM?

**Comment by kaiwalyakoparkar** [comment]
Looks good to me

**Comment by kaiwalyakoparkar** [comment]
Hey please link issue number #72

**Comment by Jeevan-Kiran-Lenka** [comment]
linked

**Pull request merged**